packages/client/tsconfig.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["./**/*"]
+  "include": ["./**/*", "../client-engine-runtime/src/interpreter/serialize.test.ts"]
```

Comment:
Why is this needed? 🤔

Reply:
uhh, looks like the language server did it when I moved the file, thanks for catching 🤦